var/spack/repos/builtin/packages/sz3/package.py

```diff
@@ -35,5 +38,16 @@ def cmake_args(self):
         return [
             "-DSZ3_USE_BUNDLED_ZSTD=OFF",
             "-DSZ3_DEBUG_TIMINGS=OFF",
+            self.define_from_variant("BUILD_MDZ", "mdz"),
             self.define_from_variant("BUILD_H5Z_FILTER", "hdf5"),
         ]
+
+    def test(self):
+        if self.spec.satisifes("@3.1.7:"):
```

Comment:
There is a typo here:  `satisifes` -> `satisfies`.

Reply:
Also, since the tests are supposed to be skipped for versions < `3.1.7` the call should be more like `self.spec.satisfies("@:3.1.6")`.

Reply:
FWIW. I  tested three different installations of the package with the above fixes and the tests ran as expected.

Reply:
fixed